<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>汉堡</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>小食</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>150</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>饮料</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>巨无霸</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡腿堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>培根蔬萃双层汉堡</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_4">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麦辣鸡翅</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>中薯条</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_6">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>麦乐鸡块</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_8">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>可乐</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButton_9">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>96</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>雪碧</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup_2</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
  <buttongroup name="buttonGroup_2"/>
 </buttongroups>
</ui>
